### DISCUSSION TITLE
unstable mode!

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8004] [QA - tech support + general]
can someone send me a screenshot of the unstable branch patch notes??? ((((DISCORD IS BLOCKED))))

**Comment by briaryoung** [comment]
Please keep topics to the same issue.

**Comment by ghost** [comment]
ok, sorry.

**Comment by ghost** [comment]
i just reopened the other issue

**Comment by ghost** [comment]
someone pls send me a screenshot though
